### ISSUE TITLE
Releasing 0.18.0

### ISSUE THREAD

**Issue by bahulneel** [typeetfunc/datascript-mori#12]
@ashnur How do we want to go about releasing 0.18.0 as I would like to start adopting it ASAP?

Maybe the thing to do is to release it on NPM scoped as @typeetfunc is responsible for the published module.

**Comment by ashnur** [comment]
I agree that it's probably better to realease it scoped. Maybe if we can move to a different build tool then we could release it under a new name or something.

**Comment by bahulneel** [comment]
I'm happy to do this unless you would rather.

**Comment by ashnur** [comment]
Please go ahead, you are probably much more familiar with the tooling. I am happy to help if needed!

**Comment by bahulneel** [comment]
OK, I've published the latest package here: https://www.npmjs.com/package/@bahulneel/datascript-mori
There's a minor version bump but that's so that the README on npm shows the correct `npm install` command.

**Comment by ashnur** [comment]
🎉  I will test it tonight as well.
